x-pack/plugins/ml/public/components/anomalies_table/anomalies_table.js

```diff
@@ -294,7 +302,7 @@ class AnomaliesTable extends Component {
     return null;
   }
 
-  toggleRow = (item) => {
+  toggleRow = (item, tab = 0) => {
```

Comment:
Minor thing, but if you click on the link to show the Examples tab, then switch to the Details tab, I wonder if clicking on the link again should then switch to the Examples tab, where currently it collapses the row?